portal/server/nunjucks-configuration/filter-localiseTimestamp.ts

```diff
@@ -0,0 +1,20 @@
+import { formatInTimeZone } from 'date-fns-tz';
+
+export const filterLocaliseTimestamp = (
+  utcTimestamp: number | string,
+  format: string,
+  targetTimezone: string,
+) => {
+  if (!utcTimestamp) {
```

Comment:
Will this ever be a case?

Reply:
It's used in `templates/contract/components/contact-overview-table.njk`, line 57 
```
{{ details.manualInclusionApplicationSubmissionDate | localiseTimestamp("dd/MM/yyyy", user.timezone) | dashIfEmpty | safe }}
```
The timestamp can be an empty string passed into `dashIfEmpty`